airflow/models/dag.py

```diff
@@ -2561,12 +2560,16 @@ def bulk_write_to_db(cls, dags: Collection["DAG"], session=NEW_SESSION):
 
         dag_ids = set(dag_by_ids.keys())
         query = (
-            session.query(DagModel)
-            .options(joinedload(DagModel.tags, innerjoin=False))
-            .filter(DagModel.dag_id.in_(dag_ids))
+            (
+                session.query(DagModel)
+                .options(joinedload(DagModel.tags, innerjoin=False))
+                .filter(DagModel.dag_id.in_(dag_ids))
+            )
+            .options(joinedload(DagModel.schedule_dataset_references))
+            .options(joinedload(DagModel.task_outlet_dataset_references))
```

Comment:
```suggestion
            session.query(DagModel)
            .options(joinedload(DagModel.tags, innerjoin=False))
            .filter(DagModel.dag_id.in_(dag_ids))
            .options(joinedload(DagModel.schedule_dataset_references))
            .options(joinedload(DagModel.task_outlet_dataset_references))
```

Looks equivalent to me? Surprising that Black allows this.

Reply:
hah yeah... not sure how that happened.